skllm/__init__.py

```diff
@@ -3,5 +3,8 @@
     MultiLabelZeroShotGPTClassifier,
     ZeroShotGPTClassifier,
 )
-from skllm.models.gpt.gpt_few_shot_clf import FewShotGPTClassifier
+from skllm.models.gpt.gpt_few_shot_clf import (
+    FewShotGPTClassifier,
+    MultiLabelFewShotGPTClassifier,
```

Comment:
We are trying to get away from top level imports (most likely they will be removed in v1.0), so it is better not to add anything new here.

Reply:
Removed